vignettes/stat_specs.Rmd

```diff
@@ -436,16 +436,33 @@ All approaches provide consistent treatment effect estimates for standard and re
+The second proposal is the so-called "information-anchored" variance which was originally proposed in the context of sensitivity analyses (@CroEtAl2019). This variance estimator is based on disentangling point estimation and variance estimation altogether. 
+The information-anchoring principle described in @CroEtAl2019 states that the relative increase in the variance of the treatment effect estimator under MAR imputation with increasing amounts of missing data should be preserved for reference-based imputation methods.
+The resulting information-anchored variance is typically very similar to the variance under MAR imputation and typically increases with increasing amounts of missing data. 
+However, the information-anchored variance does not reflect the actual variability of the reference-based estimator under repeated sampling and the resulting inference is highly conservative resulting in a substantial power loss (@Wolbers2021). 
+Moreover, to date, no Bayesian or frequentist framework has been developed under which the information-anchored variance provides correct inference for reference-based missingness assumptions, nor is it clear whether such a framework can even be developed.
+
+Reference-based conditional mean imputation (`method_condmean()`) and bootstrapped likelihood-based multiple methods (`method = method_bmlmi()`) obtain standard errors via resampling and hence target the frequentist variance (@Wolbers2021,  @vonHippelBartlett2021). 
+For finite samples, simulations for a sample size of $n=100$ per group reported in @Wolbers2021 demonstrated that conditional mean imputation combined with the jackknife (`method_condmean(type = "jackknife")`) provided exact protection of the type one error rate whereas the bootstrap (`method_condmean(type = "bootstrap")`) was associated with a small type I error inflation (between 5.1\% to 5.3\% for a nominal level of 5\%). 
+For reference-based conditional mean imputation, an alternative information-anchored variance can be obtained by following a proposal by @Lu2021. 
+The basic idea of @Lu2021 is to obtain the information-anchored variance via a MAR imputation combined with a delta-adjustment where delta is selected in a data-driven way to match the reference-based estimator. 
+For conditional mean imputation, the proposal by @Lu2021 can be implemented by choosing the delta-adjustment as the difference between the conditional mean imputation under the chosen reference-based assumption and MAR on the original dataset. 
+An illustration of how the different variances can be obtained for conditional mean imputation in `rbmi` is provided in the vignette "Frequentist and information-anchored inference for reference-based conditional mean imputation" (`vignette(topic = "CondMean_InfoAnchored", package = "rbmi")`).
```

Comment:
```suggestion
An illustration of how the different variances can be obtained for conditional mean imputation in `rbmi` is provided in the vignette "Frequentist and information-anchored inference for reference-based conditional mean imputation" (`vignette(topic = "CondMean_Inference", package = "rbmi")`).
```

I would maybe change the name of the vignette since we have included how to derive the frequentist SE as well.